### DISCUSSION TITLE
ScriptReference/Selection-activeGameObject

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#206] [Notes]
# ScriptReference/Selection-activeGameObject

https://docs.unity3d.com/ScriptReference/Selection-activeGameObject.html

**Comment by mvi** [comment]
When setting `Selection.activeGameObject` you often want to highlight the newly selected object in the hierarchy, to do this call [`EditorGUIUtility.PingObject()`](https://docs.unity3d.com/ScriptReference/EditorGUIUtility.PingObject.html):  
  

```csharp
Selection.activeGameObject = newObject;  
EditorGUIUtility.PingObject(Selection.activeGameObject);
```
